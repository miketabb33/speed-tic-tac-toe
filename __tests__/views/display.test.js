import React from 'react'
import renderer from 'react-test-renderer'
import Display from '../../views/display.js'

test('render xFirst', () => {
	const display = renderer.create(
		<Display value = { Display.xFirst } />
	)

	const result = display.toJSON()
	expect(result).toMatchSnapshot()
})

test('render xTurn', () => {
	const display = renderer.create(
		<Display value = { Display.xTurn} />
	)

	const result = display.toJSON()
	expect(result).toMatchSnapshot()
})

test('render oTurn', () => {
	const display = renderer.create(
		<Display value = { Display.oTurn} />
	)

	const result = display.toJSON()
	expect(result).toMatchSnapshot()
})

test('render xWin', () => {
	const display = renderer.create(
		<Display value = { Display.xWin} />
	)

	const result = display.toJSON()
	expect(result).toMatchSnapshot()
})

test('render oWin', () => {
	const display = renderer.create(
		<Display value = { Display.oWin} />
	)

	const result = display.toJSON()
	expect(result).toMatchSnapshot()
})