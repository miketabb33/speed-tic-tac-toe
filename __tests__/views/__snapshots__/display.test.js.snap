// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render oTurn 1`] = `
<div
  className="display"
>
  O's Turn
</div>
`;

exports[`render oWin 1`] = `
<div
  className="display"
>
  O is the Winner!
</div>
`;

exports[`render xFirst 1`] = `
<div
  className="display"
>
  X Goes First
</div>
`;

exports[`render xTurn 1`] = `
<div
  className="display"
>
  X's Turn
</div>
`;

exports[`render xWin 1`] = `
<div
  className="display"
>
  X is the Winner!
</div>
`;
